config/manager/manager.yaml

```diff
@@ -25,6 +25,19 @@ spec:
         - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},ClusterResourceSet=${EXP_CLUSTER_RESOURCE_SET:=false},ClusterTopology=${CLUSTER_TOPOLOGY:=false},RuntimeSDK=${EXP_RUNTIME_SDK:=false}"
         image: controller:latest
         name: manager
+        env:
+        - name: POD_NAMESPACE
```

Comment:
If we go ahead with this PR. Namespace can be read from `/var/run/secrets/kubernetes.io/serviceaccount/namespace` as well. Just needs a fallback so that the controller can also be run for dev purposes outside of Kubernetes